src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -1170,6 +1171,20 @@ public boolean onWireCutterRightClick(ForgeDirection side, ForgeDirection wrench
         return false;
     }
 
+    @Override
+    public boolean onSolderingToolRightClick(ForgeDirection side, ForgeDirection wrenchingSide, EntityPlayer aPlayer,
+            float aX, float aY, float aZ) {
+        if (supportsVoidProtection()) {
```

Comment:
call superclass `onSolderingToolRightClick`, even if it do nothing. it will save a lot of headache if we ever add any features to superclass involving solder right click